projects/python-daily-visualizations/readme.md

```diff
@@ -0,0 +1 @@
+Add project overview/purpose of code
```

Comment:
Please see some of our other readme files and use that as a template for your readme